.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,6 @@
 **Pull Request Summary**
 
-> Describe what changes this pull request makes to the repository
 
 **Check list**
-- [ ] contains breaking changes
-- [ ] adds new feature
-- [ ] modifies existing feature (bug fix or improvements)
-- [ ] relies on other tasks
-- [ ] documentation changes
-- [ ] tests and/or benchmarks are included
-- [ ] changed API client type definition or chain metadata
-
-**This pull request makes the following changes:**
-
-**Adds**
-- (ex: Add feature A)
-
-**Fixes**
-- (ex: Fix validation function)
-
-**Changes**
-- (ex: Change document B)
-
-**To-dos**
-> *Feel free to remove this section if it's not applicable
-
-- [ ] (ex: add user list)
+- [ ] added unit tests
+- [ ] updated documentation
```

Comment:
do we need a document update on the frame and on the new node release? (https://github.com/AstarNetwork/Astar/pull/692/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R6)

Reply:
I was thinking more about the Rustdoc actually.

Usually only runtime upgrades warrant updates in docs since they're the ones who introduce actual changes.
Perhaps we just move it under a TODO in `Astar` repo?

Reply:
Yeah, it's good to make the `Astar` repo contain all changes that the public can see, and keep the frame repo for the devs IMO.